scout/adapter/mongo/case.py

```diff
@@ -1591,14 +1591,11 @@ def update_case_sanger_variants(self, institute_obj, case_obj, case_verif_varian
 
                 else:
                     # old variant had Sanger validation ordered
-                    # check old event to collect user_obj that ordered the verification:
                     # set sanger ordered status for the new variant as well:
-                    updated_var = self.order_verification(
```

Comment:
Skipping the create a new event step. The old event should still cover the new variant no? At least the new variant still has Sanger ordered when I tested on stage;

CaseS page:

<img width="599" alt="image" src="https://github.com/user-attachments/assets/52b74682-4c6e-48c8-a5fd-bfdf1eada54c" />


Case page:

<img width="328" alt="image" src="https://github.com/user-attachments/assets/92001eab-01d9-4f72-ad08-89f2762aedac" />